### PR TITLE
Add minimal example to the hyper-schema Overview

### PR DESCRIPTION
**Pull request by handrews**
This should be enough to give a feel for the basics without
needing a detailed explanation or taking up too much space
during the introductory sections.

Addresses #473.

### CODE REVIEW

File: jsonschema-hyperschema.xml [hunk 1]
```diff
@@ -147,6 +147,38 @@
                 other link serializations or representation formats at runtime, or pre-emptively
                 follow links to facilitate server push usage.
             </t>
+            <figure>
+                <preamble>
+                    The empty schema is a valid hyper-schema, as all schema keywords are optional,
+                    and simply has no effect.  No hyperlinks are applied by default.  The following
```
Comment (philsturgeon) [inline jsonschema-hyperschema.xml]: Maybe this should be a new paragraph, to show we've moved on from "no links is fine.

Here are some links"

Comment (handrews) [reply jsonschema-hyperschema.xml]: I went back and forth on that a couple of time.  Let's see what @dlax (or anyone else who wants to jump in) says :-)

Reading your comment I'm leaning towards splitting it, but like I said, back and forth...

Comment (dlax) [reply jsonschema-hyperschema.xml]: I think I'm missing the purpose of the first two sentences. And I'm not sure I get what you want to split off either :(

Why not simply have the third sentence phrased as "Here is an example of JSON Hyper-Schema with a single link, ..."? No need to mention it's minimal or non-trivial, I think.

(Apart from this, this looks good to me.)

Comment (handrews) [reply jsonschema-hyperschema.xml]: @dlax I like the simplification idea.  The empty schema is covered elsewhere, and you're right that the non-trivial/minimal language wasn't adding anything important.

I've updated this with a shorter caption now.  @philsturgeon does this look OK to you?

**Pull request merged**